sprint/sprint-guide.md

```diff
@@ -0,0 +1,50 @@
+(sprint_guide)=
+# Sprint Guide
+
+Here are some guidelines and suggestions about the sprint for participants and volunteers!
+
+## General Information
+
+- Be Polite
+- Ask questions
+- Don't use any discriminatory terms
+- Use gender neutral pronouns unless you know someone's preferred pronouns
+- Don't use words like "guys", instead say "folks" or "people"
+- Be inclusive and kind
+
+The sprint will be happening on Zoom, the sign up instructions are available on Meetup, please try to join on the starting time so you can recieve proper help and guidance for the sprint. There will be breakout rooms inside the Zoom call that participants will be assigned to:
+- Women and Non binary folks: this breakout room is specifically for women and non binary folks, we hope you have a meaningful space here where you're heard!
+- New to open source: this breakout room is for newcomers to open source so you can get suitable help
+- Code related PR: If you are making a code related contribution then you'll be assigned to this breakout room
+- Documentation related PR: If you are making a Documentation related contribution then you'll be assigned to this breakout room
+- Spanish Speaker: If you prefer spanish over english to communicate this breakout room will be useful, Oriol Abril Pla (one of our PyMC volunteers) will be there to help you out
+
+These breakout rooms aren't rigid, you can choose which one you wish to be in, they're created solely to help us help you!
+
+## Volunteer Guide
+
+1. Breakout rooms will be created at the begining of the meeting, but do join 15 minutes before the sprint so you can get the hang of Zoom and these rooms to make things smoother.
+2. Feel free to create new breakout rooms, if needed.
+3. Be patient and kind to sprint participants, if you feel unable to help someone with a doubt or issue, redirect it to another volunteer or PyMC team member if someone is available. If that does not work, consider suggesting a new issue to the participant.
+4. Encourage people to introduce themselves and ask questions. 
+5. Do flag any(really ANY) inappropriate behaviour. Warn the participant or remove them from the Zoom meeting in these cases:
+
+    - They make a discriminatory remark
+    - They are being persistently rude
+    - They are a spammer
+   
+### Tentative structure of the Sprint
+
+- All participants join, the first 5-10 mins are spent with everyone introducing and saying hi
+- Participants are redirected to the board consisting of available issues that they can choose [from here](https://github.com/pymc-devs/pymc/projects/4)
+- If a participant doesn't find a suitable issue here they're encouraged to ask the volunteers for help, who will suggest an issue or task based on their said skills
+- Participants go to breakout rooms and work on their issues, they should make a comment on the issue they're interested in, after which volunteers will update that issue on the Sprint Dashboard for clarity.
```

Comment:
Careful here because there are several issues on which multiple people can work at the same time